modules/gdscript/gdscript_cache.h

```diff
@@ -88,7 +88,7 @@ class GDScriptCache {
 	static Ref<GDScriptParserRef> get_parser(const String &p_path, GDScriptParserRef::Status status, Error &r_error, const String &p_owner = String());
 	static String get_source_code(const String &p_path);
 	static Ref<GDScript> get_shallow_script(const String &p_path, const String &p_owner = String());
-	static Ref<GDScript> get_full_script(const String &p_path, Error &r_error, const String &p_owner = String());
+	static Ref<GDScript> get_full_script(const String &p_path, Error &r_error, const String &p_owner = String(), const bool p_update_from_disk = false);
```

Comment:
We don't use const for bool, int and float parameters.

Reply:
OK, thanks for pointing it out.